.env.example

```diff
@@ -66,7 +66,7 @@ RAILS_MAX_THREADS=5
 
 # The email from which all outgoing emails are sent
 # could user either  `email@yourdomain.com` or `BrandName <email@yourdomain.com>`
-MAILER_SENDER_EMAIL=Chatwoot <accounts@chatwoot.com>
+MAILER_SENDER_EMAIL=Julian from Cruise Control <accounts@chatwoot.com>
```

Comment:
change the sender email to be support@getcruisecontrol.com. applies to all files where we're using a chatwoot email